### PR TITLE
21w07a worldgen mappings

### PR DESCRIPTION
**Pull request by jaskarth**
Maps the majority of the worldgen classes in 21w07a, as well as a few other things used by them.

### CODE REVIEW

File: mappings/net/minecraft/util/dynamic/Codecs.mapping [hunk 1]
```diff
@@ -26,10 +26,27 @@ CLASS net/minecraft/class_5699 net/minecraft/util/dynamic/Codecs
 		ARG 1 element
 	METHOD method_32852 (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Z
 		ARG 1 element
+	METHOD method_33817 either (Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
```
Comment (liach) [inline mappings/net/minecraft/util/dynamic/Codecs.mapping]: Mind see #2084?

File: mappings/net/minecraft/world/gen/decorator/Decoratable.mapping [hunk 2]
```diff
@@ -25,5 +25,11 @@ CLASS net/minecraft/class_5432 net/minecraft/world/gen/decorator/Decoratable
 		ARG 1 maxCount
 			COMMENT the maximum repetition count
 	METHOD method_30377 rangeOf (Lnet/minecraft/class_5843;Lnet/minecraft/class_5843;)Ljava/lang/Object;
-		COMMENT Applies the {@code minecraft:range} decorator, which returns the input
-		COMMENT position with a y value from zero to {@code max}.
+		COMMENT
```
Comment (liach) [inline mappings/net/minecraft/world/gen/decorator/Decoratable.mapping]:
```suggestion
```

**Review by liach — COMMENTED** [review]
Overall looks more comprehensive than shnupbup's pr.

File: mappings/net/minecraft/util/dynamic/Codecs.mapping (hunk 1 above)
Comment (jaskarth) [reply mappings/net/minecraft/util/dynamic/Codecs.mapping]: Sure, will use that instead

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/util/dynamic/Codecs.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/util/dynamic/Codecs.mapping]: In fact if yours is merged first I can fix conflicts. Not a big problem

**Review by YanisBft — APPROVED** [review]

**Review by LambdAurora — APPROVED** [review]

**Pull request merged**